### ISSUE TITLE
Read the Docs build fails with Poetry >= 1.8.0

### ISSUE THREAD

**Issue by sethfischer** [sethfischer/rover#103]
Read the Docs build fails with Poetry >= 1.8.0.

[Read the Docs builds 23584649](https://readthedocs.org/projects/sethfischer-osr/builds/23584649/).

```
python -m sphinx -T -b html -d _build/doctrees -D language=en . $READTHEDOCS_OUTPUT/html
Running Sphinx v7.2.6
Traceback (most recent call last):
⋮
  File "/home/docs/checkouts/readthedocs.org/user_builds/sethfischer-osr/checkouts/102/docs/conf.py", line 6, in <module>
    import sphinx_rtd_theme  # type: ignore[import-untyped] # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'sphinx_rtd_theme'
```

**Comment by sethfischer** [comment]
See python-poetry/poetry#9025.

**Comment by sethfischer** [comment]
Fixed with 00380b17c0d9fd09a020e755429fafcdcc174589.
